homeassistant/util/energy.py

```diff
@@ -38,5 +38,17 @@ def convert(value: float, from_unit: str, to_unit: str) -> float:
     if from_unit == to_unit:
         return value
 
+    return _convert(value, from_unit, to_unit)
+
+
+def normalize(value: float, from_unit: str) -> float:
```

Comment:
This is bypassing all the sanity checks, can we call it `normalize_no_checks`, or add a named parameter to control if  the checks should be done or not?

Reply:
I have renamed the function to `to_normalized_unit`, and added a doctype comment to indicate that checks are bypassed.